src/Workspaces/Remote/Razor/ServiceHubServiceBase.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
```

Comment:
consider if we can move off of these in the future and use Systme.Text.Json instead.

Reply:
JSON RPC is using Newtonsoft.Json. We'll move away from directly depending on JSON RPC at some point. Not yet though.